src/dconfigfile.cpp

```diff
@@ -52,6 +52,12 @@ inline static bool isValidFilename(const QString& filename)
     QRegularExpressionMatch match = regex.match(filename);
     return match.hasMatch();
 }
+inline static bool isValidAppId(const QString& appId)
+{
+    if (appId.contains(' '))
```

Comment:
Should not contains `\n` '\t' too?

Reply:
https://github.com/linuxdeepin/dtkcore/pull/417